ckanext/dcat/tests/test_json_harvester.py

```diff
@@ -108,9 +109,9 @@ def _test_harvest_create(
         kwargs['source_type'] = 'dcat_json'
         harvest_source = self._create_harvest_source(url, **kwargs)
 
+        fq = "+type:dataset harvest_source_id:{0}".format(harvest_source['id'])
```

Comment:
Is there a reason this line was moved? I'd rather not change old tests if possible.

Reply:
Its easier to debug it in this case. 
No logic was changed by this move